travis-ci/deploy-to-npm.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -evx
+
+echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
+VERSION=$(jq --raw-output .version lerna.json)
+./node_modules/.bin/lerna publish \
+  --skip-git \
+  --repo-version "$VERSION" \
+  --yes \
+  --force-publish=* \
+  --npm-client=npm \
+  --npm-tag="mth-test-$VERSION"
```

Comment:
You probably want to remove `--npm-tag="mth-test-$VERSION"`

Reply:
For npm though we need to be able to use a tag other than latest. I was doing that based on the branch name before.